internal/scripts/installation/compile_MET_all.sh

```diff
@@ -827,19 +844,46 @@ fi
 export MET_PYTHON_BIN_EXE=${MET_PYTHON_BIN_EXE:=${MET_PYTHON}/bin/python3}
 export MET_PYTHON_LD
 export MET_PYTHON_CC
-export LDFLAGS="-Wl,--disable-new-dtags"
 
-if [[ $machine == "Mac" ]]; then
-  export LDFLAGS=""
+# add flags to user-defined LDFLAGS for MacOS
+if [[ $machine != "Mac" ]]; then
+  LDFLAGS="${LDFLAGS} -Wl,--disable-new-dtags"
 fi
 
 # https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
 # ${parameter:+word}
 # If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.
-export LDFLAGS="${LDFLAGS} -Wl,-rpath,${LIB_DIR}/lib:${MET_PROJ:+:$MET_PROJ/lib64}:${LIB_DIR}/lib${MET_NETCDF:+:$MET_NETCDF/lib}${MET_HDF5:+:$MET_HDF5/lib}${MET_BUFRLIB:+:$MET_BUFRLIB}${MET_GRIB2CLIB:+:$MET_GRIB2CLIB}${MET_PYTHON_LIB:+:$MET_PYTHON_LIB}${MET_GSL:+:$MET_GSL/lib}${ADDTL_DIR:+:$ADDTL_DIR}"
-export LDFLAGS="${LDFLAGS} -Wl,-rpath,${LIB_JASPER:+$LIB_JASPER}${LIB_LIBPNG:+:$LIB_PNG}${LIB_Z:+$LIB_Z}"
-export LDFLAGS="${LDFLAGS} ${LIB_JASPER:+-L$LIB_JASPER} ${LIB_LIBPNG:+-L$LIB_LIBPNG} ${MET_HDF5:+-L$MET_HDF5/lib} ${ADDTL_DIR:+-L$ADDTL_DIR}"
-export LIBS="${LIBS} -lhdf5_hl -lhdf5 -lz -ltiff"
+
+# add LIB_DIR/lib and LIB_DIR/lib64 to rpath and -L
+LDFLAGS="${LDFLAGS} -Wl,-rpath,${LIB_DIR}/lib -L${LIB_DIR}/lib  -Wl,-rpath,${LIB_DIR}/lib64 -L${LIB_DIR}/lib64"
+
+# if variables are set, add <VALUE>/lib to rpath and -L
+for x in $MET_CAIRO $MET_FREETYPE $MET_GSL $MET_HDF $MET_HDF5 $MET_NETCDF; do
+    arg="${x:+-Wl,-rpath,$x/lib -L$x/lib}"
+    if [[ "$LDFLAGS" != *"$arg"* ]]; then
+	LDFLAGS+=" $arg"
```

Comment:
Consistent indentation with spaces instead of tabs.
```suggestion
      LDFLAGS+=" $arg"
```